packages/components/src/ui/create-styles/create-compiler/create-compiler.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * External dependencies
+ */
+import { is } from '@wp-g2/utils';
```

Comment:
This import/dependency probably something we can avoid? (use lodash helpers or just regular JS checks)

Reply:
Whoops, yes, looks like I missed one 😅